_posts/2024-03-11-release-0_5_0.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Kroxylicious release 0.5.0"
+date:   2024-03-10 00:00:00 +0000
+author: "Robert Young"
+author_url: "https://www.github.com/robobario"
+categories:  [releases, kroxylicious, record-encryption]
+---
+
+The Kroxylicious project is very pleased to announce the release of [Kroxylicious 0.5.0](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.5.0).
+
+### Record Encryption
+
+This release was focused on refining [Record Encryption](/use-cases) to be ready for experimentation in secure environments, adding:
+* Enable users to configure Hashicorp Vault TLS, providing custom keystore and truststore
+* Enable users to supply the Hashicorp Vault token via file rather than having to inject it into the proxy configuration
+* Include Record Encryption filter (and other Kroxylicious project supported filters) in the binary distribution
+* Release a Docker image `quay.io/kroxylicious/kroxylicious` also containing the supported filters
+* Internal refactoring to control the usage of key material, preparing to support alternate Ciphers and configurable Additional Authenticated Data
+
+The protocol for immutable encrypted data written to the broker is now aligned with our initial design and we guarantee data encrypted with version 0.5.0 
+of the Filter will be decryptable forever (assuming the keys stored in the KMS are available forever for decryption).
```

Comment:
Maybe reword this slightly?
```suggestion
The protocol for immutable encrypted data written to the broker is now aligned with our initial design and we guarantee data encrypted with version 0.5.0 
of the Filter will be decryptable by all future versions of Kroxylicious (assuming the encryption keys stored in the KMS remain available).
```
I feel like this makes it a bit more clear that we don't mean decryptable by *anything*, we specifically mean future versions of Kroxylicious. CYA, etc.

Reply:
Done, though I've changed to `future versions of the Record Encryption Filter`, I figured they might evolve separately. WDYT? Too confusing to make the distinction here?

Reply:
That makes sense to me 👍